Arrowgene.Ddon.Shared/Entity/PacketStructure/S2CCraftRecipeGetCraftGradeupRecipeRes.cs

```diff
@@ -17,7 +17,7 @@ public S2CCraftRecipeGetCraftGradeupRecipeRes()
 
         public byte Category { get; set; }
         public List<CDataMDataCraftGradeupRecipe> RecipeList { get; set; }
-        public List<CDataCommonU32> UnknownItemList { get; set; } // filled with random item IDs, didn't seem to do anything?
+        public List<CDataCommonU32> UnknownItemList { get; set; }
```

Comment:
Should it still be called unknown item list? Isn't it more like "UpradableItemList"? As you said, without this the client crashes and we know it's the exact same list that's coming in from the request?